code/modules/background/citizenship/human.dm

```diff
@@ -32,7 +32,7 @@
 	backpack_contents = list(
 		/obj/item/storage/box/ceti_visa = 1,
 		/obj/item/storage/box/tcfl_pamphlet = 1,
-		/obj/item/gun/energy/pistol = 1
+		/obj/item/gun/energy/disruptorpistol = 1
```

Comment:
Should remain a pistol. No reason for the consular officer to get a nt one.

Reply:
```suggestion
		/obj/item/gun/energy/pistol = 1
```